fixtures/tracks/sports/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+Do stuff in track hints.
```

Comment:
I'd say something more explicit like: 'This is the content of the track hints file'